index.js

```diff
@@ -43,7 +43,7 @@ if (argv.gasprice < 1 || argv.gasprice > 1000) throw "--gasprice must be between
     minSponsorTokens: { rawValue: toWei("100") }, // Min sponsor position size of 100 synthetic tokens.
     liquidationLiveness: 7200, // 2 hour liquidation liveness.
     withdrawalLiveness: 7200, // 2 hour withdrawal liveness.
-    excessTokenBeneficiary: "0x54f44eA3D2e7aA0ac089c4d8F7C93C27844057BF", // UMA Store contract.
+    excessTokenBeneficiary: "0x41AF40Eb92Bec4dD8DA77103597838b3dBBD3B6f", // UMA Kovan Store contract.
```

Comment:
We can probably use `getAddress()` to grab the store address for whatever network they're using. Thoughts on doing that instead?

Reply:
Yep better approach. Added this and moved finding `networkId` before defining `empParams` to use in `getAddress()`. Lmk what you think